packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -137,4 +138,14 @@ export interface IFluidDataStoreRuntime
 	 * the data store's entryPoint.
 	 */
 	readonly entryPoint?: IFluidHandle<FluidObject>;
+
+	/**
+	 * @deprecated - Will be removed in future major release. Please migrate all usage to the "entryPoint" pattern.
```

Comment:
Maybe put a comment here mentioning these used to be included via IFluidRouter.  For whenever we do find-all for IFluidRouter in coming weeks.